vm/src/frame.rs

```diff
@@ -799,12 +827,28 @@ impl ExecutingFrame<'_> {
             bytecode::Instruction::BinaryOperationInplace { op } => {
                 self.execute_binop_inplace(vm, op.get(arg))
             }
+            bytecode::Instruction::BinarySubscript => {
+                let key = self.pop_value();
+                let container = self.pop_value();
+                self.state
+                    .stack
+                    .push(container.get_item(key.as_object(), vm)?);
+                Ok(None)
+            }
             bytecode::Instruction::LoadAttr { idx } => self.load_attr(vm, idx.get(arg)),
             bytecode::Instruction::StoreAttr { idx } => self.store_attr(vm, idx.get(arg)),
             bytecode::Instruction::DeleteAttr { idx } => self.delete_attr(vm, idx.get(arg)),
             bytecode::Instruction::UnaryOperation { op } => self.execute_unop(vm, op.get(arg)),
             bytecode::Instruction::TestOperation { op } => self.execute_test(vm, op.get(arg)),
             bytecode::Instruction::CompareOperation { op } => self.execute_compare(vm, op.get(arg)),
+            bytecode::Instruction::IsOperation(neg) => {
```

Comment:
This instruction is duplication of a part of `TestOperation`.
I expect one of them to be eliminated.

Reply:
Is `TestOperation` something that was in cpython bytecode but was removed?

We should really maintain a mapping of cpython instructions and associated instructions

Reply:
I guess so.
CPython changed a lot of instructions for recent a few years. The mapping would be useless very quickly, but better to had one.

Reply:
I think I'll consolidate instructions in later prs, this seems more ergonomic to me too.